helm_deploy/values-preprod.yaml

```diff
@@ -9,6 +9,13 @@ generic-service:
     API_BASE_URL_NOMIS: "https://api-preprod.prison.service.justice.gov.uk"
     INDEX_COMPLETE_THRESHOLD: 500000
 
+  # Additional allowlist rules
+  # See also allowlist values helm_deploy/prisoner-offender-search/values.yaml
+  allowlist:
+    prum-prod-platform-1: "18.170.20.75/32"
```

Comment:
Should be 

- 18.134.149.68
- 18.133.201.123
- 52.56.76.34

Reply:
This being preprod, are we sure you want their test env to have access to production data?